test/model/content_test.dart

```diff
@@ -943,6 +943,387 @@ class ContentExample {
+            '<span class="mord">'
+              '<span class="mord mathnormal">a</span>'
+              '<span class="msupsub">'
+                '<span class="vlist-t">'
+                  '<span class="vlist-r">'
+                    '<span class="vlist" style="height:0.8019em;">'
+                      '<span style="top:-3.113em;margin-right:0.05em;">'
+                        '<span class="pstrut" style="height:2.7em;"></span>'
+                        '<span class="sizing reset-size6 size3 mtight">'
+                          '<span class="mord mtight">'
+                            '<span class="mord mtight">′</span></span></span></span></span></span></span></span></span></span></span></span></span></p>',
+    [
+      MathBlockNode(
+        texSource: 'a\'',
+        nodes: [
+          KatexSpanNode(
+            styles: KatexSpanStyles(),
+            text: null,
+            nodes: [
+              KatexStrutNode(heightEm: 0.8019, verticalAlignEm: null),
+              KatexSpanNode(
+                styles: KatexSpanStyles(),
+                text: null,
+                nodes: [
+                  KatexSpanNode(
+                    styles: KatexSpanStyles(
+                      fontFamily: 'KaTeX_Math',
+                      fontStyle: KatexSpanFontStyle.italic),
+                    text: 'a',
+                    nodes: null),
+                  KatexSpanNode(
+                    styles: KatexSpanStyles(textAlign: KatexSpanTextAlign.left),
+                    text: null,
+                    nodes: [
+                      KatexVlistNode(
+                        rows: [
+                          KatexVlistRowNode(
+                            verticalOffsetEm: -3.113 + 2.7,
+                            node: KatexSpanNode(
+                              styles: KatexSpanStyles(marginRightEm: 0.05),
+                              text: null,
+                              nodes: [
+                                KatexSpanNode(
+                                  styles: KatexSpanStyles(fontSizeEm: 0.7),
+                                  text: null,
+                                  nodes: [
+                                    KatexSpanNode(
+                                      styles: KatexSpanStyles(),
+                                      text: null,
+                                      nodes: [
+                                        KatexSpanNode(
+                                          styles: KatexSpanStyles(),
+                                          text: '′',
+                                          nodes: null),
+                                      ]),
+                                  ]),
+                              ])),
+                        ]),
+                    ]),
+                ]),
+            ]),
+        ]),
+    ]);
+
+  static const mathBlockKatexVertical2 = ContentExample(
+    'math block katex vertical 2',
```

Comment:
Can you give these examples names that reflect what interesting situations they're each exercising that isn't covered by the other examples?

For example, this one introduces `vlist-t2`, which is definitely an interesting case that's good to be sure to cover.